src/angular/table/sort/sort-header.html

```diff
@@ -51,7 +47,8 @@
         stroke="currentColor"
         stroke-width="1"
         d="M7.5,14.25 L11.5,18.25 L15.5,14.25"
-        [@indicator]="_getArrowDirectionState()"
+        class="sbb-sort-indicator"
+        [ngClass]="[_getArrowDirectionState(), _shouldSkipTransitionIndicator() ? 'no-transition' : '']"
```

Comment:
I think we can use conditional classes as described here: https://angular.dev/api/common/NgClass